nixos/lib/test-driver/Machine.pm

```diff
@@ -43,7 +43,7 @@ sub new {
             if defined $args->{bios};
         $startCommand .= $args->{qemuFlags} || "";
     } else {
-        $startCommand = Cwd::abs_path $startCommand;
+        #$startCommand = Cwd::abs_path $startCommand;
```

Comment:
So you think the `abs_path` is probably unnecessary now? I'm not even sure anything even passes in an explicit `startCommand` other than my test. I'm thinking of trying to remove it and seeing if anything breaks. Seem reasonable?